src/fits.jl

```diff
@@ -1,11 +1,13 @@
 # helper functions
 
-#=
-FITSIO.jl takes over memory read in by cfitsio, which reads in row-major form,
-whereas when Julia takes that memory, it is assumed as column major.
-Therefore all data read by `read` is transposed.
-Related comment: https://github.com/JuliaAstro/CCDReduction.jl/pull/16#issuecomment-638492572
-=#
+"""
+    getdata(::FITSIO.ImageHDU)
+
+Loads image array form `ImageHDU`
+
+FITSIO.jl takes over memory read in by cfitsio, which reads in row-major form, whereas when Julia takes that memory, it is assumed as column major.
+Therefore all data read by [`FITSIO.read`](http://juliaastro.github.io/FITSIO.jl/latest/api.html#Base.read-Tuple{ImageHDU}) is transposed. This function allows the user to read data in a consistent way to `Array` by transposing after reading.
+"""
 function getdata(hdu::ImageHDU)
```

Comment:
`write_data` and `getdata` offers some grammatical inconsistency. I would settle on `getdata` (mimics an astropy function, is *getting* the *data* as an array from the fits file) and `writefits` (matches grammar of no underscore, is *writing* the data to a *fits* file)

little inconsistencies like this if we correct them now, hopefully we avoid burdening future developers and users!

Reply:
Sure, I will switch to `writefits`!